packages/teams-js/src/public/chat.ts

```diff
@@ -0,0 +1,119 @@
+import { sendAndHandleStatusAndReason as sendAndHandleError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { FrameContexts } from '../public/constants';
+import { runtime } from '../public/runtime';
+
+/**
+ *
+ * @internal
+ */
+interface OpenChatRequest {
+  /**
+   * @hidden
+   * The message to send when opening chat
+   */
+  message?: string;
+}
+
+/**
+ * @hidden
+ * Hide from docs.
+ * ------
+ *
+ * @internal
+ */
+export interface OpenSingleChatRequest extends OpenChatRequest {
+  /**
+   * @hidden
+   * User's UPN to open chat with
+   */
+  user: string;
+}
+
+/**
+ * @hidden
+ * Hide from docs.
+ * ------
+ *
+ * @internal
+ */
+export interface OpenGroupChatRequest extends OpenChatRequest {
+  /**
+   * @hidden
+   * Array containing UPNs of users to open chat with
+   */
+  users: string[];
+  /**
+   * @hidden
+   * The display name of a conversation for 3 or more users
+   */
+  topic?: string;
+}
+
+/**
+ * @hidden
+ * Namespace to interact with the conversational subEntities inside the tab
+ *
+ * @alpha
+ */
+export namespace chat {
+  /**
+   * @hidden
```

Comment:
If this is public, do we still want to hide documentation from the official docs?

Reply:
Note: this applies to all the @hidden annotations

Reply:
Ahhh. I see in the PR description you plan to tackle these later. Carry on. 🚀